SmartEnrollmentFor911/SmartEnrollmentFor911/Pages/SearchCrimes.cshtml.cs

```diff
@@ -15,23 +15,31 @@ public class SearchCrimesModel : PageModel
         public bool searchFinished { get; set; }
         [BindProperty]
         public long zipSearch { get; set; }
-        public QuickType.CrimeIncidents[] crimes;
-        public QuickType911Enrollment.Smart911Enrollment[] enrollments;
-        public QuickType.CrimeIncidents[] crimesFiltered;
-        public QuickType911Enrollment.Smart911Enrollment[] enrollmentsFiltered;
-        public void OnGet()
-        {
+        public CrimeIncident.CrimeIncidents[] crimes;
+        public Smart911Enrollments.Smart911Enrollment[] enrollments;
+        public CrimeIncident.CrimeIncidents[] crimesFiltered;
+        public Smart911Enrollments.Smart911Enrollment[] enrollmentsFiltered;
+        public string crimeJsonEndPoint = "https://data.cincinnati-oh.gov/resource/k59e-2pvf.json";
+        public string smart911EnrollmentJsonEndPoint = "https://data.cincinnati-oh.gov/resource/rtu7-isj6.json";
 
+        //Reusable method to het the input Json Streams
+        public string GetDataFromJson(string endpoint)
+        {
+            using (WebClient webClient = new WebClient())
+            {
+                return webClient.DownloadString(endpoint);
+            }
         }
-        public void OnPost()
+        public void OnGet()
```

Comment:
Are you sure you want to change this to OnGet from OnPost?  OnGet doesn't honor [BindProperty], unless you specifically tell it to.